doc/modules/changes/20180427_gassmoeller

```diff
@@ -0,0 +1,5 @@
+Changed: The default value for AdditionalSeismicOutputs was changed from -1 to
+a signaling Nan. This ensures every plugin that creates seismic outputs needs
+to use a material model that actually fills those outputs.
+<br>
+(Rene Gassmoeller, 2018/04/27)
```

Comment:
Would it be worth adding this documentation to the `seismic_anomalies` post-processor description? Alternatively, is there already an assert that produces a similar error message when the values are filled with Nan?